backend/src/src-bdpimport/src/test/java/com/bosch/osmi/sw360/bdp/datasink/thrift/ThriftUploaderTest.java

```diff
@@ -140,7 +141,7 @@ public void testIfIsAlreadyInDbWrapperEmpty() {
 
     @Test
     public void testIfIsAlreadyInDbWrapperEmptyList() {
-        Optional<String> toCheck = thriftUploader.searchExistingEntityId(Optional.of(Collections.EMPTY_LIST),
```

Comment:
(from discussion) so this is about best practice to prefer method over constant access? (fine, just for clarification)